plugins/channel_track.go

```diff
@@ -399,7 +399,7 @@ func (p *ChannelTracker) addUserToChannel(b *seabird.Bot, user, channel string)
 	if u == nil {
 		u = &User{
 			Nick:     user,
-			UUID:     uuid.NewV4().String(),
+			UUID:     uuid.Must(uuid.NewV4()).String(),
```

Comment:
This can technically cause a panic at runtime, but it should be *very* rare that this fails.